content/2021/2021-02/2021-02-08-pulsar-weekly.md

```diff
@@ -0,0 +1,217 @@
+
+- [Broker] Add the streaming dispatcher.
+
+    https://github.com/apache/pulsar/pull/9056 (@[MarvinCai](https://github.com/MarvinCai))
+
+- [Broker] Handle bad-version in metadata-store if the broker updates the ZooKeeper metadata from the ZooKeeper client.
+
+    https://github.com/apache/pulsar/pull/9412 (@[rdhabalia](https://github.com/rdhabalia))
+
+- [Broker] Add alerts for expired/expiring tokens.
+
+    https://github.com/apache/pulsar/pull/9321 (@[RobertIndie](https://github.com/RobertIndie))
+
+- [Test] Add the test for creating producers for non-persistent replicated topics.
+
+    https://github.com/apache/pulsar/pull/9322 (@[congbobo184](https://github.com/congbobo184))
+
+- [Functions] Support setting the memory limit for the Pulsar client used in the `ThreadRuntime` in Pulsar Functions
+
+    https://github.com/apache/pulsar/pull/9320 (@[jerrypeng](https://github.com/jerrypeng))
+
+- [Functions] Optimize the built-in source/sink startup process by eliminating the redundant NAR unpacking and checksum calculation.
+
+    https://github.com/apache/pulsar/pull/9413 (@[jerrypeng](https://github.com/jerrypeng))
+
+- [Admin] Support schema REST API for V1 topics.
+
+    https://github.com/apache/pulsar/pull/9218 (@[rdhabalia](https://github.com/rdhabalia))
+
+- [Functions] Enhance the Kubernetes manifest customizer with default options.
+
+    https://github.com/apache/pulsar/pull/9445 (@[freeznet](https://github.com/freeznet))
+
+- [Managed Ledger] Support configuring the `ManagedLedger` storage.
+
+    https://github.com/apache/pulsar/pull/9397 (@[sijie](https://github.com/sijie))
+
+- [Admin] Ensure that the admin operations on Statestore are not blocked.
+
+    https://github.com/apache/pulsar/pull/9348 ([@pkumar-singh](https://github.com/pkumar-singh))
+
+- [C++] Remove the Boost::System runtime dependency.
+
+    https://github.com/apache/pulsar/pull/9498 (@[merlimat](https://github.com/merlimat))
+
+- [CLI] Update the `-cluster` parameter in t`he delete-cluster-metadata` command to `--cluster`.
+
+    https://github.com/apache/pulsar/pull/9487 (@[yangl](https://github.com/yangl))
+
+- [Topic Policy] Support getting the applied policy for retention.
+
+    https://github.com/apache/pulsar/pull/9362 (@[315157973](https://github.com/315157973))
+
+- [Topic Policy] Support getting the applied policy for the offloader.
+
+    https://github.com/apache/pulsar/pull/9505 (@[315157973](https://github.com/315157973))
+
+### Notable Bug Fix
+
+- [Flaky Test] Fix the `MessagePublishBufferThrottleTest` Flaky test.
+
+    https://github.com/apache/pulsar/pull/9376 (@[Renkai](https://github.com/Renkai))
+
+- [Flaky Test] Fix the `TopicReaderTest.testMultiReaderIsAbleToSeekWithTimeOnMiddleOfTopic` Flaky test.
+
+    https://github.com/apache/pulsar/pull/9375 (@[Renkai](https://github.com/Renkai))
+
+- [C++] Fix the issue that pulsar-client-java does not set `replicator_to` into messages when batching is enabled.
```

Comment:
`Fix broken replicated msg to specific cluster`.

BTW, if you think the original description is not clear, you can just change `pulsar-client-java` here to `C++ client` or `pulsar-client-cpp`.